bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -363,20 +363,15 @@ async def deactivate_infraction(
         actor = infraction["actor"]
         type_ = infraction["type"]
         id_ = infraction["id"]
-        inserted_at = infraction["inserted_at"]
-        expiry = infraction["expires_at"]
 
         log.info(f"Marking infraction #{id_} as inactive (expired).")
 
-        expiry = dateutil.parser.isoparse(expiry).replace(tzinfo=None) if expiry else None
-        created = time.format_infraction_with_duration(inserted_at, expiry)
-
         log_content = None
         log_text = {
             "Member": f"<@{user_id}>",
             "Actor": f"<@{actor}>",
             "Reason": infraction["reason"],
-            "Created": created,
+            "Created": time.format_with_duration(infraction["inserted_at"], infraction["expires_at"]),
```

Comment:
When manually unbanning someone with a perma ban, this raises the below error. I assume this is due to expires_at being `None`
```
bot_1       | Traceback (most recent call last):
bot_1       |   File "/usr/local/lib/python3.9/site-packages/discord/ext/commands/core.py", line 85, in wrapped
bot_1       |     ret = await coro(*args, **kwargs)
bot_1       |   File "/bot/bot/exts/moderation/infraction/infractions.py", line 270, in unban
bot_1       |     await self.pardon_infraction(ctx, "ban", user)
bot_1       |   File "/bot/bot/exts/moderation/infraction/_scheduler.py", line 292, in pardon_infraction
bot_1       |     log_text = await self.deactivate_infraction(response[0], send_log=False, notify=notify)
bot_1       |   File "/bot/bot/exts/moderation/infraction/_scheduler.py", line 374, in deactivate_infraction
bot_1       |     "Created": time.format_with_duration(infraction["inserted_at"], infraction["expires_at"]),
bot_1       |   File "/bot/bot/utils/time.py", line 303, in format_with_duration
bot_1       |     duration = humanize_delta(timestamp, other_timestamp, max_units=max_units)
bot_1       |   File "/bot/bot/utils/time.py", line 196, in humanize_delta
bot_1       |     start = arrow.get(args[1]) if len(args) == 2 else arrow.utcnow()
bot_1       |   File "/usr/local/lib/python3.9/site-packages/arrow/api.py", line 80, in get
bot_1       |     return _factory.get(*args, **kwargs)
bot_1       |   File "/usr/local/lib/python3.9/site-packages/arrow/factory.py", line 228, in get
bot_1       |     raise TypeError("Cannot parse argument of type None.")
bot_1       | TypeError: Cannot parse argument of type None.
```